docs/widget.rst

```diff
@@ -17,10 +17,24 @@ In Adminsite
     from leaflet.admin import LeafletGeoAdmin
 
     from .models import WeatherStation
+    from .models import WeatherSensor
 
-
+    # A Basic Example
     admin.site.register(WeatherStation, LeafletGeoAdmin)
 
+   """A More Complicated Example
+      Notice ModelAdmin(admin.ModelAdmin) also has to change to modelAdmin(LeafletGeoAdmin)
+      @admin.register() decorator not required for Django-Leaflet, but is for inlines.
```

Comment:
Don't you mean the other way around? register is required for the `TabularInline` but is for the `LeafletGeoAdmin`?

Reply:
No. I tested this. The decorator is required for the inline only.

Reply:
Then there is something I don't understand with what you wrote. The `LeafletGeoAdmin` is correctly decorated but does it have to do with the inlines?

Reply:
The Decorator is not required for django-leaflet, but _if_ you have an inline it is required.

Reply:
It should be required with or without inline (or `admin.site.register`).